requirements/base.txt

```diff
@@ -0,0 +1,17 @@
+albumentations==1.1.0
+attrdict==2.0.1
+defusedxml==0.7.1
+kornia==0.5.6
+lxml==4.6.3
+matplotlib==3.4.3
+networkx==2.5.1
+nncf==2.0.0
+numpy==1.19.5
```

Comment:
Can we change this to `numpy~=1.19.2` and the `scikit-image` to `scikit-image>=0.17.2`. Tox passes after these changes.